### PR TITLE
 Changes to be committed(testing exercise 2.2):

### PR DESCRIPTION
**Pull request by daokid-ai**
Created two files to  create a diff state in the repo in my branch "richard-aicamp-branch"
        new file:   hello_world.py
	new file:   test_branch_file.txt

### CODE REVIEW

File: hello_world.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#This is a file to test the github exercise and have a python file.
+print("Hello World")
```
Comment (zikegcwk) [inline hello_world.py:2]: nice! 

We will however not merge this in as it has nothing to do with the detector. 

This comment will show you how to do a code review, as we should for our students.

Comment (daokid-ai) [reply hello_world.py:2]: Cool!

Comment (daokid-ai) [reply hello_world.py:2]: I should close the pull request at this point then?

**Comment by zikegcwk** [comment]
Yes please!

So getting to this step you have essentially the whole process of fork a
repo, edit some code, and submit a PR. That's what we should do for our
students as they will need to collaborate using this.

**Pull request closed without merging**